### PR TITLE
Xenolove: Zaddat RIGs, Phoronoid RIGs soon to come

### PR DESCRIPTION
**Pull request by JennaSilverShark**
YlOseajfuP
dreammaker_Q47q8f3du7

dreammaker_2HONvwcMGo
dreammaker_5h4P3UUxOq
dreammaker_j9YNGrCb0j
dreammaker_6YGe3e8KAX

### CODE REVIEW

**Comment by Zandario** [comment]
omg these are hot

File: code/modules/clothing/spacesuits/rig/rig_pieces_vr.dm [hunk 1]
```diff
@@ -12,36 +12,29 @@
...
-		SPECIES_ZORREN_FLAT		= 'icons/mob/species/fennec/suit_vr.dmi',
-		SPECIES_ZORREN_HIGH 	= 'icons/mob/species/fox/suit_vr.dmi',
-		SPECIES_VULPKANIN		= 'icons/mob/species/vulpkanin/suit.dmi',
-		SPECIES_PROMETHEAN		= 'icons/mob/species/skrell/suit.dmi',
-		SPECIES_XENOHYBRID		= 'icons/mob/species/unathi/suit.dmi',
-		SPECIES_VOX 			= 'icons/mob/species/vox/suit.dmi',
-		SPECIES_TESHARI = 'icons/mob/species/teshari/suit.dmi'
-		)
+		SPECIES_ZADDAT 			= 'icons/mob/species/zaddat/shoes.dmi'
+		)//Fingers crossed it will reference back to the base sheet if it can't find the others
```
Comment (Zandario) [inline code/modules/clothing/spacesuits/rig/rig_pieces_vr.dm:24]: The base list isn't complete, please copy them over

File: code/modules/clothing/spacesuits/rig/rig_pieces.dm [hunk 2]
```diff
@@ -42,6 +43,10 @@
 	species_restricted = null
 	gender = PLURAL
 	icon_base = null
+	sprite_sheets = list(
+		SPECIES_ZADDAT = 'icons/mob/species/zaddat/shoes.dmi'
+		) //Zaddat Engi RIG appearance is unique
+
 	force = 5 // if you're kicking someone with something meant to keep you locked on a hunk of metal...
 
 /obj/item/clothing/suit/space/rig
```
Comment (Zandario) [inline code/modules/clothing/spacesuits/rig/rig_pieces.dm:52]: I meant to say copy them here, the other set is a override

**Comment by JennaSilverShark** [comment]
Closing for Merge Conflict

**Pull request closed without merging**